link/kprobe.go

```
@@ -193,6 +196,7 @@ func kprobe(symbol string, prog *ebpf.Program, opts *KprobeOptions, ret bool) (*
 	if opts != nil {
 		args.cookie = opts.Cookie
 		args.offset = opts.Offset
+		args.maxactive = opts.Maxactive
```

Please return an error if `opts.Maxactive < 0` here.